api/src/models/website.model.js

```diff
@@ -28,6 +28,14 @@ const websiteSchema = new mongoose.Schema(
       type: String,
       required: false,
     },
+    layout: {
+      type: String,
+      required: false,
+    },
+    theme: {
```

Comment:
Also, in terms of naming, themeOption vs theme? I think the former implies that there are a preset number of themes to choose from. Same for the layouts

Reply:
Concretely, I think we can rename to layoutOption/themeOption as they are enums

Reply:
I think making it `layoutOption` and `themeOption` would make it consistent with `publicationOptions` but I'm not fussed about whichever you choose.

Reply:
`publicationOptions` is named as such (and I agree in this case) because it contains two sub-fields, `sortBy` and `groupBy`. This is not the case for theme and layout. Also, we're using `sortBy` and not `sortByOptions`, unless you guys think that should be changed as well.

Reply:
I think the layout and theme are somehow referring outlook of the website, maybe we can combine them together into 1 variable and having the layout and theme be the sub-fields, similar to the publicationOptions?

Reply:
Okay I'm fine with that